src/v3/src/types/schema.ts

```diff
@@ -254,7 +254,10 @@ export interface FieldElement extends UISchemaElement {
    */
   showAsterisk?: boolean;
   options: {
-    inputMeta: Input;
+    // TODO: remove customLabel after https://github.com/okta/okta-auth-js/pull/1422
```

Comment:
https://github.com/okta/okta-auth-js/pull/1422 was merged, can we remove this now if we get a release of auth-js done?

Reply:
Yes, I'll ask Shuo to release new version of okta-auth-js and will use it in SIW